timely/src/dataflow/operators/generic/builder_raw.rs

```diff
@@ -161,6 +161,26 @@ impl<G: Scope> OperatorBuilder<G> {
         let inputs = self.shape.inputs;
         let outputs = self.shape.outputs;
 
+        // Generate a tracing span for the function.
+        #[cfg(feature = "tracing")]
+        let span = {
+            let location = std::panic::Location::caller();
+            // timely.console.span is a marker target allowing us to
+            // turn these spans on and off specifically
+            // TODO(guswynn): check with tracing folks if there is a better
+            // way to do this
+            tracing::trace_span!(
+                target: "timely.console.span",
```

Comment:
in general, `tracing` targets are Rust module paths when they're not explicitly overrididen; so `::` is typically used as a separator rather than `.` --- the use of `.` is not _forbidden_ by `tracing`, but it's not the general convention.

i don't really understand the role of the `timely::console::span` target --- i would probably just make this 
```suggestion
            tracing::trace_span!(
                target: "timely::dataflow::operator",
```
or something if you don't want to use the full module path (the user probably doesn't care about `::generic::builder_raw` here?). you'll still be able to filter that target out without having a weird target for it.

Reply:
I want to have a set target so that if this code ever moves, set filters in our downstream binary don't break, but good point, `timely::dataflow::operator` is probably better!